src/test/java/wolox/bootstrap/BootstrapApplicationTests.java

```diff
@@ -3,10 +3,14 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.context.junit4.SpringRunner;
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
+@PropertySource(value = "classpath:application.properties")
```

Comment:
Is this necessary? Doesn't springboot already fetch this one automatically?

Reply:
I just checked this, the application context doesn't load without this annotation :(

Reply:
Probably it's necessary because we are not using the Spring Boot v2.2.6
Let's try to delete this line after @darkBasic666 update the Spring Boot version :)